### PR TITLE
Update how storage cost is calculated

### PR DESCRIPTION
**Pull request by rfornea**
-Properly calculates the PRL invoice
-Adds unit tests

### CODE REVIEW

File: models/upload_sessions_test.go [hunk 1]
```diff
@@ -295,3 +296,94 @@ func (ms *ModelSuite) Test_GetTreasureIndexes() {
...
+	}
+
+	vErr, err := u.StartUploadSession()
+	invoice := u.GetInvoice()
+	ms.Nil(err)
+	ms.Equal(0, len(vErr.Errors))
+
+	// expecting to be charged for 2 full sectors
+	// should be same price as previous test
+	ms.Equal(decimal.NewFromFloatWithExponent(0.09375000000000001, -16), invoice.Cost)
```
Comment (rfornea) [inline models/upload_sessions_test.go]: The 2nd parameter in NewFloatWithExponent is the exponent, and if you give it `decimal.NewFromFloatWithExponent(0.09375, -16)` it ignores the -16 and the exponent will be -5.  And when you do `decimal.NewFromFloatWithExponent(0.0937500000000000, -16)` it truncates back to -5.  Hence the additional 1, which gets cut off but it makes the result have the right exp value.

File: models/upload_sessions.go [hunk 2]
```diff
@@ -68,6 +66,8 @@ const (
 	TreasureBuried
 )
 
+var StoragePeg = 64 // GB per year per PRL; TODO: query smart contract for real storage peg
```
Comment (pzhao5) [inline models/upload_sessions.go]: Or you could make it decimal number rather than int

Comment (rfornea) [reply models/upload_sessions.go]: Okay, I'll change it.

**Comment by rfornea** [comment]
Making a small update to this in the morning

**Pull request merged**